src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+// Actions
+const STATUS_CHECKED = 'STATUS_CHECKED';
```

Comment:
- [ ]  Kindly match the duck pattern, it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS` Kindly update it.